src/main/java/Deadline.java

```diff
@@ -6,9 +6,14 @@ public Deadline(String description, String by) {
         this.by = by;
     }
 
+    public Deadline(String description, boolean done, String by) {
+        super(description, done);
+        this.by = by;
+    }
+
     @Override
     public String toString() {
-        return "[D]" + super.toString() + " (by: " + by + ")";
+        return "D | " + super.toString() + " | by: " + by;
```

Comment:
Perhaps consider using `String.format()` instead of just concatenating strings. It may be much concise. For your consideration 😊